flexmeasures/data/services/scheduling.py

```diff
@@ -213,10 +242,11 @@ def create_scheduling_job(
 
 
 def make_schedule(
-    sensor_id: int,
```

Comment:
Breaking change. Instead, we should keep it and add:
```
if sensor_id is not None:
    current_app.logger.warning(
        "The `sensor_id` keyword argument is deprecated. Please, consider using the argument `asset_or_sensor_id`."
    )
    asset_or_sensor_id = sensor_id
    entity_type = "sensor"
```

Reply:
Indeed!